#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_toyfontface_type_base.dpatch by Torsten Marek <shlomme@debian.org>
##
## DP: Set base type of ToyFontFace correctly.

@DPATCH@
diff -urNad pycairo-1.8.4~/cairo/cairomodule.c pycairo-1.8.4/cairo/cairomodule.c
--- pycairo-1.8.4~/cairo/cairomodule.c	2009-03-19 09:43:58.000000000 +0100
+++ pycairo-1.8.4/cairo/cairomodule.c	2009-03-24 00:14:50.000000000 +0100
@@ -173,7 +173,7 @@
     PycairoFontFace_Type.tp_base = &PyBaseObject_Type;
     if (PyType_Ready(&PycairoFontFace_Type) < 0)
         return;
-    PycairoToyFontFace_Type.tp_base = &PyBaseObject_Type;
+    PycairoToyFontFace_Type.tp_base = &PycairoFontFace_Type;
     if (PyType_Ready(&PycairoToyFontFace_Type) < 0)
         return;
     PycairoFontOptions_Type.tp_base = &PyBaseObject_Type;
